compiler/src/codegen/compcore.re

```diff
@@ -3418,37 +3432,9 @@ let compile_globals = (wasm_mod, env, {globals} as prog) => {
 };
 
 let compile_main = (wasm_mod, env, prog) => {
-  let preamble =
-    if (Env.is_runtime_mode()) {
```

Comment:
One more use of runtime mode nixed, how close are we to killing it? 😉

Reply:
Basically, runtime mode exists because we have code that needs to run before a memory manager exists, so the answer to this is once we move over to wasm GC.